### PR TITLE
Added a WhatWeDo page explaining what the company does, also made a t…

### PR DESCRIPTION
**Pull request by Gregamoo**
Created a What we do sectionand added the information onto it.

### CODE REVIEW

File: src/components/Footer/Footer.styles.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const StyledFooter = styled.div`
   @media screen and (min-width: 768px) {
     display: flex;
     flex-wrap: wrap;
+    padding: 1rem 2.5rem;
```
Comment (emilybuc) [inline src/components/Footer/Footer.styles.js:14]: Why did you change this?

Comment (Gregamoo) [reply src/components/Footer/Footer.styles.js:14]: I made it so that the footer padding was consistent with the rest of the content on the web page as that is what the design showed :)

**Review by emilybuc — APPROVED** [review]
